src/styles/codeStyles/index.android.js

```diff
@@ -0,0 +1,23 @@
+const codeWordWrapper = {
+    height: 10,
+};
+const codeWordStyle = {
+    borderLeftWidth: 0,
+    borderRightWidth: 0,
+    borderTopLeftRadius: 0,
+    borderBottomLeftRadius: 0,
+    borderTopRightRadius: 0,
+    borderBottomRightRadius: 0,
```

Comment:
Isn't this just borderRadius: 0? Why does this have to be included?

Reply:
@jasperhuangg these style props take precedence over `borderRadius`. Due to we need to specify `borderRadius` we can't remove it from the main code styles but on native, the new implementation only wants border radius for the first and last word. Thus to override it, I have to be specific for each edge.
In Short, `borderRadius` does not work.